web/integration_test.go

```diff
@@ -284,10 +284,10 @@ func TestIntegration_ExternalAdapter_Pending(t *testing.T) {
 	cltest.CreateBridgeTypeViaWeb(t, app, bridgeJSON)
 	j = cltest.FixtureCreateJobViaWeb(t, app, "../internal/fixtures/web/random_number_bridge_type_job.json")
 	jr := cltest.CreateJobRunViaWeb(t, app, j)
-	jr = cltest.WaitForJobRunToPend(t, app.Store, jr)
+	jr = cltest.WaitForJobRunToPendExternal(t, app.Store, jr)
```

Comment:
Shouldn't this be `cltest.WaitForJobRunToPendBridge`?

Reply:
Amended commit and force pushed